include/lemlib/movements/purepursuit.hpp

```diff
@@ -23,7 +25,7 @@ class PurePursuit : public Movement {
          * @param reversed whether the chassis should move forwards (false) or backwards (true)
          * @param maxSpeed the maximum speed the robot can move at. 127 by default
          */
-        PurePursuit(float trackWidth, const asset& path, float lookaheadDist, int timeout, bool reversed, int maxSpeed);
+        PurePursuit(float trackWidth, Path path, float lookaheadDist, int timeout, bool reversed, int maxSpeed);
```

Comment:
```suggestion
        PurePursuit(float trackWidth, Path& path, float lookaheadDist, int timeout, bool reversed, int maxSpeed);
```
Copy the path by reference